modules/tinymce/src/core/main/ts/html/ParserUtils.ts

```diff
@@ -6,7 +6,9 @@ import Schema, { SchemaMap } from '../api/html/Schema';
 
 const paddEmptyNode = (settings: DomParserSettings, args: ParserArgs, blockElements: SchemaMap, node: AstNode): void => {
   if (args.insert && blockElements[node.name]) {
-    node.empty().append(new AstNode('br', 1));
+    const astNode = new AstNode('br', 1);
+    astNode.attr('data-mce-bogus', '1');
```

Comment:
Why was this needed, this could potentially break things?

Reply:
Because the initial data did not have the bogus-attribute added as they apparently should have. Tests seems to be passing as expected.

Reply:
Right so it's only when it's being inserted and that empty block will still be treated as empty and get padded on export.